### ISSUE TITLE
Resiliency Algorithms

### ISSUE THREAD

**Issue by mohanr** [aws/aws-sdk-java#2353]
Pluggable algorithms using https://github.com/resilience4j/resilience4j

## Describe the Feature
There is a backoff strategy built into the JMS library. More algorithms for resiliency from a library
like https://github.com/resilience4j/resilience4j are needed.

## Additional Context
There may be some support for Resiliency4J  that I am not aware of.

## Your Environment

* AWS Java SDK version used:
* JDK version used:
* Operating System and version:

**Comment by debora-ito** [comment]
Hi @mohanr thank you for reaching out. The SDK for Java 1.11.x still supports Java 7 so this feature will be more suitable for SDK for Java 2.x, which is built on top of Java 8. 

Do you want me to transfer this feature request to the Java SDK 2.x [repository](https://github.com/aws/aws-sdk-java-v2)? Would it help in your use case?

**Comment by mohanr** [comment]
Yes. Are there some guidelines for adoption of such features and contribution ?
